docs/astro/src/content/docs/reference/elements/text.mdx

```diff
@@ -32,10 +32,13 @@ export component TextExample inherits Window {
 
 A `Text` element for displaying text.
 
-If the `width` and `height` properties are not set, the text will
-be expand in a single line. If the `wrap` property is set along with
-a `width` and `height`, the text will be wrapped into multiple lines
-of text.
+By default, the `min-width`, `min-height`, `preferred-width`, and `preferred-height`
+of a `Text` element are set to fit the full text as if it were displayed on a single line
+(unless the text contains explicit line breaks).
+However, if either the `wrap` or `overflow` properties are set to `word-wrap` or `elide`,
```

Comment:
This sounds like I can set `wrap` to `elide` and `overflow` to `word-wrap`...
Suggestion:

    However, if the `wrap` property is set to `word-wrap`, and/or if the `overflow` property is set to `elide`,

Reply:
```suggestion
However, if the `wrap` property is set to `word-wrap`, and/or if the `overflow` property is set to `elide`,
```